socialhome/streams/app/tests/fixtures/AuthorBar.fixtures.js

```diff
@@ -1,19 +1,18 @@
 import _ from "lodash/core"
+import faker from "faker"
 
 
 const getAuthorBarPropsData = function (args = {}) {
     return _.defaults({}, args, {
-        handle: "",
-        name: "",
-        guid: "",
-        currentBrowsingProfileId: "",
-        homeUrl: "",
-        absoluteUrl: "",
-        imageUrlSmall: "",
+        handle: faker.internet.email(),
```

Comment:
Use `faker.internet.exampleEmail()` to make a safe email that will not accidentally deliver anywhere.